R/default-scoring-rules.R

```diff
@@ -0,0 +1,155 @@
+#' @title Select Scoring Rules From A List of Possibilities
+#' @description Helper function to return only the scoring rules selected by
```

Comment:
If this is a helper function for users why isn't it exported?

Reply:
I thought of it as an internal helper function that's used to reduce the amount of duplicated code. It's just the internal implementation of the exclude/include functionality, so not sure users would ever need it. Do you think there is a use case?

Reply:
I think when it comes to combining and creating custom lists there is likely a use case and personally would prefer to call the function containing the defaults and then pipe into the selection function I think

Reply:
Good with me. Are you fine with keeping both options? I.e. allowing both
```
rules_quantile() |>
  select_rules("wis")
```
and
```
rules_quantile("wis")
```

Or do you think the `select` and `exclude` arguments should be removed from `rules_*()`?

Reply:
I think I like having both because it allows you to 
a) compose your scores as you want
b) do something like `score(data, rules_quantile("wis")` in a concise way

Reply:
I'm fine with having both for sure.